### DISCUSSION TITLE
Virtual workshop on development with Rocket

### DISCUSSION THREAD

**Discussion by szabgab** [rwf2/Rocket#2798] [Show & Tell]
I've started to run a series of online (virtual) workshops. I've just [scheduled one for Rocket](https://www.meetup.com/code-mavens/events/301294669/).
While I know enough of Rocket to provide an introductory workshop, but I am far from being an expert. I think it would be really nice if some of the more knowledgeable people would also attend the event and correct me if necessary and help with the questions. See the [event](https://www.meetup.com/code-mavens/events/301294669/)

**Comment by the10thWiz** [comment]
@szabgab I'm not sure if I'm an expert, but I know quite a bit.

**Reply by szabgab** [reply]
You'd be welcome
